optimization.py

```diff
@@ -128,10 +132,16 @@ def perturb_bacilli(node, config, imageshape):
     # push the new cell over the previous in the node
     node.push(Bacilli(cell.name, x, y, width, length, rotation))
 
+def split_proba(length):
+    """Returns the split probability given the length of the cell."""
+    # Determined empirically based on previous runs
+    MEAN = 17
```

Comment:
I think we changed this, but after you had submitted this pull request. May as well update it before I accept the PR.